<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWidget</class>
 <widget class="QWidget" name="DataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="topPanel" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="topPanelLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Dataset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbDataset">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbChallenge">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="cmbProtocol">
        <item>
         <property name="text">
          <string>coco</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>scikitlearn</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbLoadDataset">
        <property name="text">
         <string>Load dataset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbLoadSession">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load session</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSaveSession">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save session</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPublish">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomPanel" native="true">
     <layout class="QVBoxLayout" name="bottomPanelLayout">
      <item>
       <widget class="QWidget" name="mapAndPlotPanel" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="widget_9" native="true">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTreeWidget" name="dataTreeWidget">
              <property name="minimumSize">
               <size>
                <width>340</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="uniformRowHeights">
               <bool>true</bool>
              </property>
              <column>
               <property name="text">
                <string>Section</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_10" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QPushButton" name="pbDeleteMulti">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbMoveTo">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Move To</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbEditLabelMulti">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Edit Label</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="bottomMargin">
             <number>30</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QWidget" name="widget_8" native="true">
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <widget class="QWidget" name="infoWidget" native="true">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>170</height>
                     </size>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                     <property name="labelAlignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                     <property name="formAlignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>Name:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLabel" name="lblInstancelName">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Label</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLabel" name="lblInstanceLabel">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>URI</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QLabel" name="lblInstanceURI">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Width</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="1">
                      <widget class="QLabel" name="lblInstanceWidth">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="0">
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Height</string>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="1">
                      <widget class="QLabel" name="lblInstanceHeight">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item row="5" column="0">
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Date Captured</string>
                       </property>
                      </widget>
                     </item>
                     <item row="5" column="1">
                      <widget class="QLabel" name="lblInstanceDate">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QWidget" name="paintingWidget" native="true">
                    <layout class="QVBoxLayout" name="paintingLayout"/>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="styleSheet">
               <string notr="true">height:2px</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="labelsGroup">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Labels</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <widget class="QWidget" name="scrallAreaWidget">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>218</width>
                 <height>668</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="labelButtonsLayout">
                <property name="leftMargin">
                 <number>12</number>
                </property>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
